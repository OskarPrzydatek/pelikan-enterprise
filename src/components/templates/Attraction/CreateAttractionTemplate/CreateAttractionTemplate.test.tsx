import { render } from '@testing-library/react';
import { describe, test, vi } from 'vitest';

import { CreateAttractionTemplateHelper } from './CreateAttractionTemplateHelper';

const mockOnSubmitCreateAttraction = vi.fn();

describe('CreateAttractionTemplate', () => {
  test('component snapshot', () => {
    const view = render(
      <CreateAttractionTemplateHelper
        onSubmitCreateAttraction={mockOnSubmitCreateAttraction}
      />
    );
    expect(view).toMatchSnapshot();
  });
});
