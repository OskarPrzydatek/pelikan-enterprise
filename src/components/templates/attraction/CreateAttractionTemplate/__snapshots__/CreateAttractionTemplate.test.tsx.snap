// Vitest Snapshot v1

exports[`CreateAttractionTemplate > component snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-hhOBVt hIPxwk"
      >
        <form
          class="sc-cwSeag qbDdz"
        >
          <h2
            class="sc-dkrFOg kaKqDq"
          >
            Dodaj atrakcję
          </h2>
          <div
            class="sc-hBxehG dvBsjd"
          >
            <div
              class="sc-fnGiBr rTdrx"
            >
              <input
                class="sc-fEXmlR hyVGRi"
                data-testid="create-attraction-name"
                id="name"
                name="name"
                placeholder="Nazwa atrakcji"
              />
            </div>
          </div>
          <div
            class="sc-hBxehG dvBsjd"
          >
            <div
              class="sc-fnGiBr rTdrx"
            >
              <input
                class="sc-fEXmlR hyVGRi"
                data-testid="create-attraction-price"
                id="price"
                name="price"
                placeholder="Cena"
              />
            </div>
          </div>
          <div
            class="sc-csuSiG hHGxKx"
          >
            <div
              class="sc-eDWCr cBAmxE"
            >
              <textarea
                class="sc-ksBlkl dmriCq"
                data-testid="attraction-description"
                id="create-description"
                name="create-description"
                placeholder="Opis"
              />
            </div>
          </div>
          <input
            class="sc-lllmON ieVIbh"
            type="submit"
            value="Dodaj"
          />
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-hhOBVt hIPxwk"
    >
      <form
        class="sc-cwSeag qbDdz"
      >
        <h2
          class="sc-dkrFOg kaKqDq"
        >
          Dodaj atrakcję
        </h2>
        <div
          class="sc-hBxehG dvBsjd"
        >
          <div
            class="sc-fnGiBr rTdrx"
          >
            <input
              class="sc-fEXmlR hyVGRi"
              data-testid="create-attraction-name"
              id="name"
              name="name"
              placeholder="Nazwa atrakcji"
            />
          </div>
        </div>
        <div
          class="sc-hBxehG dvBsjd"
        >
          <div
            class="sc-fnGiBr rTdrx"
          >
            <input
              class="sc-fEXmlR hyVGRi"
              data-testid="create-attraction-price"
              id="price"
              name="price"
              placeholder="Cena"
            />
          </div>
        </div>
        <div
          class="sc-csuSiG hHGxKx"
        >
          <div
            class="sc-eDWCr cBAmxE"
          >
            <textarea
              class="sc-ksBlkl dmriCq"
              data-testid="attraction-description"
              id="create-description"
              name="create-description"
              placeholder="Opis"
            />
          </div>
        </div>
        <input
          class="sc-lllmON ieVIbh"
          type="submit"
          value="Dodaj"
        />
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
